zcash_client_backend/src/address.rs

```diff
@@ -273,8 +293,12 @@ mod tests {
 
         let transparent = { None };
 
+        #[cfg(feature = "orchard")]
         let ua = UnifiedAddress::from_receivers(orchard, sapling, transparent).unwrap();
 
+        #[cfg(not(feature = "orchard"))]
+        let ua = UnifiedAddress::from_receivers(sapling, transparent).unwrap();
```

Comment:
This could just be
```rust
let ua = UnifiedAddress::from_receivers(
    #[cfg(feature = "orchard")] orchard,
    sapling,
    transparent,
).unwrap();
```
(which will be simpler once we add the "sapling" feature).